var/spack/repos/builtin/packages/openmx/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import glob
+
+
+class Openmx(MakefilePackage):
+    """OpenMX (Open source package for Material eXplorer) is a software
+       package for nano-scale material simulations based on density functional
+       theories (DFT), norm-conserving pseudopotentials, and pseudo-atomic
+       localized basis functions."""
+
+    homepage = "http://www.openmx-square.org/index.html"
+    url      = "http://t-ozaki.issp.u-tokyo.ac.jp/openmx3.8.tar.gz"
+
+    version('3.8', sha256='36ee10d8b1587b25a2ca1d57f110111be65c4fb4dc820e6d93e1ed2b562634a1')
+
+    resource(name='patch',
+             url='http://www.openmx-square.org/bugfixed/18June12/patch3.8.5.tar.gz',
+             sha256='d0fea2ce956d796a87a4bc9e9d580fb115ff2a22764650fffa78bb79a1b30468',
+             placement='patch',
+             when='@3.8')
+
+    depends_on('mpi')
+    depends_on('fftw')
+    depends_on('blas')
+    depends_on('lapack')
+
+    parallel = False
+
+    phases = ['edit', 'build']
+
+    def edit(self, spec, prefix):
+        # Move contents to source/
+        # http://www.openmx-square.org/bugfixed/18June12/README.txt
+        patch_files = []
+        patch_files = glob.glob('./patch/*')
+        mv = which('mv')
+        for f in patch_files:
+            mv(f, './source/')
```

Comment:
Spack has a builtin `copy` method you can use.

Reply:
Thank you for your help!
I changed `mv` to the builtin `copy` method.